integ-tests/src/main/java/com/google/base/TestBase.java

```diff
@@ -49,4 +55,38 @@ public static long getTimestampByDate(String dateString) {
       }
     }
   }
+
+  /**
+   * @param dbList List of extracted from DB items
+   * @param avroList List of uploaded from Avro items
+   * @param sqlPath Path to an SQL request that queries from DB
+   * @param avroFilePath Path to exported Avro file
+   */
+  public static void assertListsEqual(
+      final List dbList, final List avroList, String sqlPath, String avroFilePath) {
+    StringBuilder dbListOutput = new StringBuilder().append('\n');
+    StringBuilder avroListOutput = new StringBuilder().append('\n');
+
+    dbList.forEach(e -> dbListOutput.append(e.toString()));
```

Comment:
Wouldn't a list of `["foo", "bar"]` evaluate to `"foobar"`. Guava's [joiner](https://guava.dev/releases/16.0/api/docs/com/google/common/base/Joiner.html) might make this easier to handle.

Reply:
Done.